turtlebot3c-nav-snap/snap/local/mux_select_key_vel.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+rosservice call /mux/select "/key_vel"
```

Comment:
I am not sure this is going to work in the context of a command-chain. Might have to use the `$SNAP` env var

Reply:
The ROS environment should be sourced. I'll double-check that it works :+1: .